src/util/dev/builder.ts

```diff
@@ -42,11 +46,19 @@ const isLogging = new WeakSet<ChildProcess>();
 let nodeBinPromise: Promise<string>;
 
 async function getNodeBin(): Promise<string> {
-  const runtime = await initializeRuntime('nodejs8.10');
-  if (!runtime.cacheDir) {
-    throw new Error('nodejs8.10 runtime failed to initialize');
+  let nodeBin: string | undefined;
+  try {
+    nodeBin = await which('node');
+  } catch (err) {
+    if (err.message.startsWith('not found')) {
```

Comment:
Can't this check for a error code?

Reply:
I simplified it using the `nothrow` option.